e2e/playwright/onboarding-tests.spec.ts

```diff
@@ -27,7 +27,7 @@ test.describe('Onboarding tests', () => {
       },
```

Comment:
Oops do we need to merge the back button onboarding PR first?

Reply:
Tested and approved #5296 @franknoirot

Reply:
My bad I branched off that branch accidentally, I've amended the history now